code/modules/reagents/chemistry/machinery/smoke_machine.dm

```diff
@@ -106,6 +106,7 @@
 		if("power")
 			on = !on
 			if(on)
+				message_admins("[key_name_admin(usr)] activated a smoke machine that contains [english_list(reagents.reagent_list)] at [ADMIN_COORDJMP(src)].")
 				log_admin("[key_name(usr)] activated a smoke machine that contains [english_list(reagents.reagent_list)] at [COORD(src)].")
```

Comment:
Unrelated to this PR, but how many fucking log_admin/message admin pairs have this exact fucking format. @optimumtact @nfreader

Reply:
all of them; it's almost like we should make a handler for it

Reply:
Logging needs a serious overhaul, but that's a topic for another time

Reply:
Please do not ping me for pointless reasons

Reply:
@optimumtact